### ISSUE TITLE
Suggestion - keep pane open even after clicking close tab

### ISSUE THREAD

**Issue by SpitFire-666** [moosingin3space/wksp-for-firefox#4]
(sorry if wrong area - new to github) When you click on the X to close a tab from the list, the list disappears. Would be nice to have the list stay open after the tab closes so you could close more tabs/switch to other tabs.

**Comment by moosingin3space** [comment]
Is this particular to the "close tab" action, or would you want this behaviour whenever you use the mouse to interact with the palette?

**Comment by moosingin3space** [comment]
(Also, thanks for tagging your issues with "Suggestion", although GitHub itself offers issue labels. Looks like I'll have to enable them.)

**Comment by SpitFire-666** [comment]
Yeah particular to the "Close Tab" action. Eg, search for a bunch of tabs related to a keyword, then close each one.

**Comment by moosingin3space** [comment]
I'll add this as a default option, just so the current behaviour can be preserved.

**Comment by benintech** [comment]
+1 here
Have you already add this option ? I can find it... (only option I have is the choice of palette hotkey)
